src/Dev/Install/Installer.php

```diff
@@ -440,7 +441,7 @@ public function createHtaccess()
 
     # Deny access to vendor, unless you're requesting main.php
     # Not restricting to the start of the path to support RewriteBase
-    RewriteCond %{REQUEST_URI} !/vendor/silverstripe/framework/main\.php
+    RewriteCond %{REQUEST_URI} !^/vendor/silverstripe/framework/main\.php
```

Comment:
what does the added `^` do?

Reply:
I just ensures vendor is at the start of the path.

Reply:
If anyone can point me to a way to make that work with `RewriteBase` I'm happy to add it back in :)

Reply:
I've got it working without rewritebase.